operatorhub-resources/api-operator/api-operator.package.yaml

```diff
@@ -18,4 +18,6 @@ packageName: api-operator
 channels:
   - name: stable
     currentCSV: api-operator.v1.2.2
-defaultChannel: stable
+  - name: 2.0.0-stable
```

Comment:
Would it be better to name this as the `stable` channel and have the other one as `1.2.x-stable` or something similar because in the future, our focus would be mostly on the newer versions of the api-operator rather than the 1.2.x versions?

Reply:
Yes, it is nice, if we can rename the previous channel. We have to check with the operator hub that, is it possible to rename a previous channel. Because if a customer is subscribed to this channel `stable` already, then there may be an unexpected behaviour, I will check this.